sdk/go/google/accesscontextmanager/v1/accessLevel.go

```diff
@@ -59,29 +59,9 @@ func GetAccessLevel(ctx *pulumi.Context,
 
 // Input properties used for looking up and filtering AccessLevel resources.
 type accessLevelState struct {
-	// A `BasicLevel` composed of `Conditions`.
```

Comment:
@pgavlin, these changes to state are expected, right?

Reply:
Yes.